ApplyToBecomeInternal/ApplyToBecomeInternal/wwwroot/package-lock.json

```diff
@@ -1,6 +1,2180 @@
 {
+	"name": "wwwroot",
```

Comment:
👀 we should revert this unless we all decide to move to the latest version of node, which we should sort out at some point